yt/docs/en/_includes/user-guide/data-processing/spyt/cluster/configuration.md

```diff
@@ -49,6 +49,18 @@ spark-launch-yt \
   --spark-cluster-version '1.36.0'
 ```
 
+### Spark configuration
+
+When using `spark-launch-yt` to configure a cluster, `--params '{"spark_conf"={...};}` is available as an option:
+
+```bash
+spark-launch-yt \
+  --proxy <cluster-name> \
+  --discovery-path my_discovery_path \
+  --params '{"spark_conf"={"spark.sql.shuffle.partitions":1,"spark.cores.max":1,"spark.executor.cores"=1};}' \
+  --spark-cluster-version '1.36.0'
```

Comment:
1.36.0 is too old SPYT version, and `--spark-cluster-version` is renamed to `--spyt-version` in recent versions of SPYT

Reply:
fix